zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -572,7 +573,78 @@ private long getUnixTimeLastRunParagraph(Paragraph paragraph) {
     return lastRunningUnixTime;
   }
 
-  public List<Map<String, Object>> getJobListforNotebook(boolean needsReload,
+  public List<Map<String, Object>> getJobListforNotebookFromParagraphId(String paragraphID) {
+    String gotNoteId = null;
+    List<Note> notes = getAllNotes();
+    for (Note note : notes) {
+      Paragraph p = note.getParagraph(paragraphID);
+      if (p != null) {
+        gotNoteId = note.getId();
+      }
+    }
+    return getJobListforNotebookFromNotebookId(gotNoteId);
+  }
+
+  public List<Map<String, Object>> getJobListforNotebookFromNotebookId(String notebookID) {
```

Comment:
Thank you for your feedback.
I expect this notebook to be changed to List in the future.
Therefore, the method name is specified as `notebook`.
But this is actually `notebook` is a single object.
I would change the method name as your opinion.
It seems that there is no confusion.
Thank you.